browser-test/src/trusted_intermediary.test.ts

```diff
@@ -54,9 +54,20 @@ test.describe('Trusted intermediaries', () => {
       lastName: 'last',
       dobDate: '1870-07-11',
     }
-    await tiDashboard.createClient(client)
-    await tiDashboard.expectDashboardNotContainClient(client)
+    await page.getByRole('button', {name: 'Add new client'}).click()
+    await waitForPageJsLoad(page)
+
+    await page.fill('#email-input', client.emailAddress)
+    await page.fill('#first-name-input', client.firstName)
+    await page.fill('#middle-name-input', client.middleName)
+    await page.fill('#last-name-input', client.lastName)
+    await page.fill('#date-of-birth-input', client.dobDate)
+
+    await page.getByRole('button', {name: 'Save'}).click()
     await validateScreenshot(page, 'dashboard-add-client-invalid-dob')
```

Comment:
nit (non-blocking):  Could we remove the word "dashboard" from the screenshot name now that the screenshot is of a separate "add client" page?